models/tmp/stg_mailchimp__lists_tmp.sql

```diff
@@ -0,0 +1,3 @@
+{{ config(enabled=var('using_automations', True)) }}
```

Comment:
This isn't an `automation` type model. We should probably not include this functionality here.